.github/workflows/ci-cd.yml

```diff
@@ -131,9 +134,40 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           target: runtime-base
-          push: true
-          registry_user: ${{ secrets.ECOSYSTEM_SERVICE_USER_DOCKER_HUB }}
-          registry_token: ${{ secrets.ECOSYSTEM_ACCESS_TOKEN_DOCKER_HUB }}
+          push: false
+          load: true
+      - name: Get EAP from base image
```

Comment:
You can still have EAP_FILE as an output from this step, no need to have it as an env. (See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter )

Reply:
Do you think it is better to have it as an output or an env?